dts/arm/st/f1/stm32f100XX.dtsi

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ * Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+ *
+ * SoC device tree include for STM32F100xx SoCs
+ * where 'xx' is replaced with the specific SoC model, {C,R,V}{4,6,8,B}
```

Comment:
This is not exact. You're introducing 8/128 variant, which is 100Xb.dtsi.
Btw, it would be cleaner to add a stm32f100.dtsi then a package file 100Xb.dsti.

Reply:
That is true. I actually had some issues wrapping my head around the naming scheme from the beginning, and this is something left behind from the start of the porting process. Since I did not find anything to break out into a `stm32f100.dtsi` (ADC seems to be included from `stm32f1.dtsi`, as noted below), I just renamed it and fixed the comments. Feel free to point out if I have missed anything.